### DISCUSSION TITLE
wiki/help-pages/loader-mods/

### DISCUSSION THREAD

**Discussion by giscus** [PrismLauncher/prismlauncher.org#676] [Wiki]
# wiki/help-pages/loader-mods/

An Open Source Minecraft launcher with the ability to manage multiple instances, accounts and mods. Focused on user freedom and free redistributability.

https://prismlauncher.org/wiki/help-pages/loader-mods/

**Comment by JettHawk1** [comment]
Im on mac, prism launcher version: 9.1, and I'm getting this error message when I try to install a mod (it won't even show what version the mod is on): A network error occurred. Could not load project info: One or more subtasks failed

**Reply by mmooggeenn** [reply]
I have the same issue

**Comment by Ivan-Sigachev** [comment]
I haven't played Minecraft for a long time, but today I decided to give it a try. The launcher JUST WOULD NOT START. THIS IS NOT THE FIRST TIME!! I took it down and installed the zip package (when I installed the bootloader in the same folder, it didn’t work either). How can I solve this problem once and for all?

**Comment by santdev332** [comment]
This new version of the launcher sucks! full of bugs. I can't start my Minecraft and I can't even edit my instances, I was stuck with version 8.4 of the Prism Launcher but I made the mistake of updating, I can't play my instances and much less can I edit them. Honestly, I'm a user who is extremely disappointed with the launcher, I haven't found anywhere how to fix this bug, not even uninstalling and installing again, the error persists and directly affects the way I play, it affects me so much that I can't even open the game...

**Comment by Minecraft64-creator** [comment]
Hello PrismLauncher Team I‘m from german. I play minecraft on my mac. My task is: I need a java software for playing minecraft simple voice chat with Prismlauncher 

Thanks
Minecraft64-creator

**Reply by Minecraft64-creator** [reply]
GuMo Minecraft64-creator du brauchst kein java

**Comment by LuigiBosca312** [comment]
when i try to launch minecraft from prism, it doesn´t iniciate, then a play wiythout conection and it doesn´t let me play the realm

**Comment by Nupey69** [comment]
Updated the launcher and now it crashes when launching my world. gives me this, Process crashed with exitcode -1073741515.

**Reply by Nupey69** [reply]
I tried everything and nothing worked....

**Comment by Worton1720** [comment]
Is there an analog to the versions folder in Prism Launcher?

**Comment by Waterking-maker** [comment]
I use Linux Mint, Prism Launcher is on Version 9.2.

This Error message goes for every instance, with and without mods.
Everytime I click on any instance, it says "It looks like we couldn't launch after 3 tries. Do you want to continue trying?".
Nothing is documented in the log. It just doesn't start.

**Comment by hannahchinyere** [comment]
When opening prism launcher it just doesn’t properly load minecraft. It either doesn’t open at all, does open but gets stuck on mojang screen, or is loading but gives me an graphics initalizing error.

**Comment by Silver-Guy** [comment]
im on steam deck and when i try to download mods i get an error message saying "please download a mod loader first" i am wondering where to download a "mod loader"

**Reply by sidesw1pe** [reply]
I'm on PC and seeing this message. I had a poke around in various places in the app but couldn't find anything to do with "mod loader". It is interesting, that the wiki specifically named "loader-mods", does not explain anything about it.

**Reply by sidesw1pe** [reply]
Well I found it. When adding an instance, you can select the mod loader there. If you've already created the instance, you will find it, intuitively (not), when editing the instance and selecting the "Version" tab. From there, on the right, you will see "Install Loader". I just picked the first one that came up whatever it was. 

I wanted to install the "Conquest Reforged" mod pack but it wasn't found. So I added all of the mod loaders but still it wasn't found. So I just gave up and downloaded it from modrinth and added the file into the mods list of the instance.

**Comment by anonymouseee-lab** [comment]
is there a way where i can combine 2 mods together and launch that so i can have 2 mods in one instance?
